src/CommunityToolkit/Xamarin.CommunityToolkit/Extensions/VisualElementExtension.shared.cs

```diff
@@ -29,5 +30,26 @@ public static void AbortAnimations(this VisualElement element, params string[] o
 			foreach (var name in otherAnimationNames)
 				element.AbortAnimation(name);
 		}
+
+		public static bool TryFindParentElementWithParentOfType<T>(this VisualElement element, out VisualElement result, out T parent) where T : VisualElement
```

Comment:
Does this have to be public? Can we make it internal? This is not something we want to promote people doing I guess

Reply:
No problem) I thought it would be useful in some cases.
But I will amend both methods to be internal

Reply:
Updated